provisioning/service/src/Config/X509CAReferences.cs

```diff
@@ -42,11 +42,6 @@ public class X509CAReferences
         [JsonConstructor]
```

Comment:
You'd need to clean up the param note about it not being allowed to be null or empty.